package.json

```diff
@@ -4,12 +4,14 @@
   "description": "Proxy that handles interactions with pagoPA",
   "main": "index.ts",
   "repository": "https://github.com/teamdigitale/italia-pagopa-proxy",
-  "author": "Sergio Manetta <sergio.manetta@it.ibm.com>",
+  "author": "Federico Feroldi <federico@teamdigitale.governo.it>",
```

Comment:
the proper way would be to have an `AUTHORS` file, add both of us and put "See AUTHORS" here